x-pack/plugins/security_solution/public/management/components/policies_selector/policies_selector.test.tsx

```diff
@@ -0,0 +1,120 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { I18nProvider } from '@kbn/i18n/react';
+import { render, act, fireEvent } from '@testing-library/react';
+import React from 'react';
+import { EndpointDocGenerator } from '../../../../common/endpoint/generate_data';
+
+import { PoliciesSelector, PoliciesSelectorProps } from '.';
+
+let onChangeSelectionMock: jest.Mock;
+
+describe('Policies selector', () => {
+  beforeEach(() => {
+    onChangeSelectionMock = jest.fn();
+  });
+  const generator = new EndpointDocGenerator('policy-list');
+  const policy = generator.generatePolicyPackagePolicy();
+  policy.name = 'test policy A';
+  policy.id = 'abc123';
+  const getElement = (params: Partial<PoliciesSelectorProps>) => {
```

Comment:
Could you set these on every test run so that we avoid mutable state being used between tests?